.github/workflows/ci.yaml

```diff
@@ -23,7 +30,5 @@ jobs:
         env:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-      - name: docker build
```

Comment:
still need to build the image, unless these are combined into a single job

Reply:
fixed!